server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/main/java/org/apache/james/webadmin/vault/routes/DeletedMessagesVaultRoutes.java

```diff
@@ -315,13 +317,27 @@ private VaultAction getVaultAction(String actionString) {
                 Joiner.on(",").join(VaultAction.plainValues()))));
     }
 
-    private void validatePurgeAction(VaultAction requestedAction) {
-        if (requestedAction != VaultAction.PURGE) {
-            throw ErrorResponder.builder()
-                .statusCode(HttpStatus.BAD_REQUEST_400)
-                .type(ErrorResponder.ErrorType.INVALID_ARGUMENT)
-                .message("Not 'purge' action")
-                .haltError();
-        }
+    private void validateAction(VaultAction requestedAction, List<VaultAction> validateActions) {
```

Comment:
`List<VaultAction> validateActions` should be `List<VaultAction> validActions`

Reply:
And we can replease List with a more fluent:  `VaultAction... validActions`

Here and everywhere : this makes caller life easier.